### DISCUSSION TITLE
Incorrect sorting with nordic characters (å,ä,ö)

### DISCUSSION THREAD

**Discussion by schoier** [microsoft-search/pnp-modern-search#3294] [Q&A]
Hi, I'm having problems with sort by refinable string mapped to crawled property Title.
I'm using a version 3 search result webpart.
Swedish characters comes in wrong order, should be last but are sorted as if they were 'a' (instead of å,ä) and 'o' (instead of ö).
![image](https://github.com/microsoft-search/pnp-modern-search/assets/33417064/1e286fb2-224a-4ab7-8d8e-61829bc6a8d6)

**Comment by kasperbolarsen** [comment]
Hi @schoier, we can only test and assist with version 4.latest. Can you test if the issue exists on version 4.9.3 ?

**Reply by schoier** [reply]
Thanks for the quick respons :) 
Not at the moment. Didn't mean to treat this as a bug, I thought it might be closer related to general settings (serch functionality) than the actual web part settings.
But I will try to test it in the newer version.

**Reply by kasperbolarsen** [reply]
I think you are right, most likely it has nothing to do with the web parts, but is an API issue.  I can  test it as well,  being a dane 😊

**Reply by wobba** [reply]
We cannot control how sorting on sortable/refinable properties work in the API unfortunately.

**Reply by schoier** [reply]
Thanks for reply @wobba! Does it mean there´s nothing we can do, or just that that's how the pnp modern search works?

**Reply by schoier** [reply]
@kasperbolarsen I've noow seen the exact same issue with standard property Filename.

**Reply by schoier** [reply]
@wobba Is it any different in v 4? Can we use graph instead? We will only show files in the search result.

**Reply by wobba** [reply]
Not how PnP Modern search works, how search API's work - and any application using them. I don't have any information either on sorting internally and what can work based on locales unfortunately.

**Reply by widarodd** [reply]
I have the same problem with the Norwegian language. We are using the last version of PnP Modern Search.
Have you found any solution @schoier ?

**Reply by wobba** [reply]
@widarodd there is no API solution.. you may add re-sorting in your UX on the results you get back. That's the best you can do. And we already do this. See https://github.com/microsoft-search/pnp-modern-search/issues/1063 where labels are ordered using JavaScript's `toLocaleLowerCase`for filters. 

For result sets you would need to get the full result set, and re-order in the UX.
